src/news/2012-10-19-plans-for-floe-year-three.md

```diff
@@ -0,0 +1,27 @@
+---
+title: Plans For FLOE Year Three
+date: '2012-10-19'
+---
+The Floe project is now in its third year. Our focus moving forward will be to advance the
+work we've been doing on personalized learning. During the first two years of the Floe project, we
+
+- developed the [Inclusive Learning Design Handbook (ILDH)](http://handbook.floeproject.org/);
+
```

Comment:
Please remove the empty lines between all the list items, it's putting the text into paragraphs which is incorrect